README.md

```diff
@@ -99,6 +99,55 @@ documentation.
 
 # Usage
 
+### Influence Functions
+
+For influence computation, follow these steps:
+
+1. Wrap your model and loss in a `TorchTwiceDifferential` object
+2. Compute influence factors by providing training data and inversion method
+
+Using the conjugate gradient algorithm, this would look like:
+```python
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from pydvl.influence import TorchTwiceDifferentiable, compute_influences
+from pydvl.influence.inversion import InversionMethod
+
+nn_architecture = nn.Sequential(
+    nn.Conv2d(in_channels=5, out_channels=3, kernel_size=3),
+    nn.Flatten(),
+    nn.Linear(27, 3),
+)
+loss = nn.MSELoss()
+influence_model = TorchTwiceDifferentiable(nn_architecture, loss)
```

Comment:
Sorry for nitpicking, but isn't `influence_model` a misnomer? One is computing influences for the model, not a model of influences. I would just leave it as `influence`.
```suggestion
model = TorchTwiceDifferentiable(nn_architecture, loss)
```

Reply:
Done